pkgs/types/src/lib.zig

```diff
@@ -12,7 +12,7 @@ pub const ValidatorIndex = u64;
 pub const Bytes48 = [48]u8;
 
 //update signature size to 4000 after ssz is fixed
-pub const SIGSIZE = 40;
+pub const SIGSIZE = 4000;
```

Comment:
fyi: size was reverted to 40 in main to test the risc0 with other stf changes that are going in